qcodes/instrument_drivers/Lakeshore/Lakeshore_model_335.py

```diff
@@ -0,0 +1,63 @@
+from typing import Dict
+
+import pyvisa as visa
+
+from .lakeshore_base import LakeshoreBase
+from .Lakeshore_model_336 import (
+    LakeshoreModel336Channel,
+    LakeshoreModel336CurrentSource,
```

Comment:
We have been slowly converting to exposing the drivers as a public api at a level above this. So could you add the driver to the `__init__` file in this folder (following the pattern of the rest) and make this file private (start the name with an underscore_